.az/mshost.yaml

```diff
@@ -98,7 +98,7 @@ jobs:
 
   - job: MacOSX
     pool:
-      vmImage: 'macOS-latest'
+      vmImage: 'macOS-12'
```

Comment:
did this framework have to be built under macOS-12?

Reply:
I think the framework itself doesn't, but the OrtExtensionsUsage project didn't build successfully with an older version of Xcode. Maybe it's because I created it with Xcode 14.1. So I just updated the CI build to use 14.1 too (in xcode-build-and-test-steps.yml).